### PR TITLE
Real Cert fix

### PR DESCRIPTION
**Pull request by mwevill**

### CODE REVIEW

File: azure/template.json [hunk 1]
```diff
@@ -92,6 +92,7 @@
         {
             "apiVersion": "2017-05-10",
             "name": "app-service-certificate",
+            "resourceGroup": "[parameters('sharedEnvResourceGroup')]",
```
Comment (jack-education) [inline azure/template.json]:
```suggestion
            "resourceGroup": "[parameters('sharedManagementResourceGroup')]",
```

File: azure/template.json [hunk 2]
```diff
@@ -107,7 +108,10 @@
                         "value": "[parameters('sharedKeyVaultName')]"
                     },
                     "keyVaultResourceGroup": {
-                        "value": "[parameters('sharedEnvResourceGroup')]"
+                        "value": "[parameters('sharedManagementResourceGroup')]"
+                    },
+                    "serverFarmId": {
```
Comment (jack-education) [inline azure/template.json]: The serverFarmId property isn't needed

File: azure/template.json (hunk 1 above)
Comment (mwevill) [reply azure/template.json]: No we are deploying to the shared env resource group from the keyvault in the management group.  matches this example https://github.com/SkillsFundingAgency/das-login-service/blob/master/azure/template.json#L70

File: azure/template.json (hunk 2 above)
Comment (mwevill) [reply azure/template.json]: Removed

**Pull request merged**